### PR TITLE
fix test failure due to devfile-registry 

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this:**
/kind tests

**What does this PR do / why we need it:**

**Which issue(s) this PR fixes:**

Fixes #

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Comment by valaparthvi** [comment]
```sh
  [FAIL] odo devfile registry command tests [It] Should list python specifically
  /go/odo_1/tests/helper/helper_generic.go:279

  content of path "0.displayName" should contain "Flask" but is "Python"
  Expected
      <string>: Python
  to contain substring
      <string>: Flask
  In [It] at: /go/odo_1/tests/helper/helper_generic.go:279

  [FAIL] odo init interactive command tests [It] should start downloading starter project only after all interactive questions have been asked
  /go/odo_1/tests/helper/helper_interactive.go:111

  ? Select language:  [Use arrows to move, type to filter]
  > .NET
    Go
    Java
    JavaScript
    PHP
    Python
    TypeScript
  dotnet
  ? Select language: d  [Use arrows to move, type to filter]
  ? Select language: do  [Use arrows to move, type to filter]
  ? Select language: dot  [Use arrows to move, type to filter]
  ? Select language: dotn  [Use arrows to move, type to filter]
  ? Select language: dotne  [Use arrows to move, type to filter]
  ? Select language: dotnet  [Use arrows to move, type to filter]
  
  ======================
  Unexpected error:
      <*xpty.errPassthroughTimeout | 0xc000542030>: {
          error: <*errors.errorString | 0xc000542010>{
              s: "passthrough i/o timeout",
          },
      }
      passthrough i/o timeout
  occurred
  In [It] at: /go/odo_1/tests/helper/helper_interactive.go:111
```
Anand, can you please fix these tests?

**Comment by anandrkskd** [comment]
> can you please fix these tests?

As discussed in yesterday stand up call we decided to not update dotnet tests as new changes on devfile-registry related to the same is breaking odo, @kadel was going to revert the changes on devfile-registry side.

cc: @kadel

**Comment by feloy** [comment]
@anandrkskd 
I think there is still this error to cover:

```
â€¢ [FAILED] [8.231 seconds]
odo devfile registry command tests
/go/odo_1/tests/integration/cmd_devfile_registry_test.go:12
  [It] Should list python specifically
  /go/odo_1/tests/integration/cmd_devfile_registry_test.go:69

[...]

  content of path "0.displayName" should contain "Flask" but is "Python"
  Expected
      <string>: Python
  to contain substring
      <string>: Flask
  In [It] at: /go/odo_1/tests/helper/helper_generic.go:279
```

**Comment by valaparthvi** [comment]
/override windows-integration-test/Windows-test
1 failure related to .NET.

**Comment by feloy** [comment]
/override Kubernetes-Integration-Tests/Kubernetes-Integration-Tests
1 failure related to .NET.

**Comment by feloy** [comment]
/override OpenShift-Integration-tests/OpenShift-Integration-tests
1 failure related to .NET. + failures related to PVC

**Comment by rm3l** [comment]
The .NET issues should be fixed with https://github.com/devfile/registry/pull/137, no?

**Comment by rm3l** [comment]
Ah, I see that they have reverted the `projectType` from `.NET` to `dotnet`. But the language is still `.NET`, which, IMO, we should handle as such.

**Comment by feloy** [comment]
The difference could be because we are using a proxy for the Devfile. I cleaned up the caches (by restarting the proxy pods). 
Restarting the tests...

File: tests/integration/interactive_init_test.go [hunk 1]
```diff
@@ -338,12 +338,12 @@ var _ = Describe("odo init interactive command tests", func() {
 		})
 	})
 
-	It("should start downloading starter project only after all interactive questions have been asked", func() {
+	FIt("should start downloading starter project only after all interactive questions have been asked", func() {
```
Comment (feloy) [inline tests/integration/interactive_init_test.go:341]: Please remove Focus

**Review by feloy — CHANGES_REQUESTED** [review]

**Pull request merged**